ecl/resource_ecl_provider_connectivity_tenant_connection_request_v2.go

```diff
@@ -0,0 +1,274 @@
+	if err != nil {
+		return fmt.Errorf("error creating ECL Provider Connectivity client: %s", err)
+	}
+
+	request, err := tenant_connection_requests.Get(client, d.Id()).Extract()
+	if err != nil {
+		return CheckDeleted(d, err, "Provider Connectivity Tenant Connection Request")
+	}
+	log.Printf("[DEBUG] Retrieved Provider Connectivity Tenant Connection Request %s: %+v", request.ID, request)
+
+	d.SetId(request.ID)
+	d.Set("status", request.Status)
+	d.Set("name", request.Name)
+	d.Set("description", request.Description)
+	d.Set("tags", request.Tags)
+	d.Set("tenant_id", request.TenantID)
+	d.Set("name_other", request.NameOther)
+	d.Set("description_other", request.DescriptionOther)
+	d.Set("tags_other", request.TagsOther)
+	d.Set("tenant_id_other", request.TenantIDOther)
+	d.Set("network_id", request.NetworkID)
+	d.Set("approval_request_id", request.ApprovalRequestID)
+
+	return nil
+}
+
+func resourceProviderConnectivityTenantConnectionRequestV2Update(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	client, err := config.providerConnectivityV2Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("error creating ECL Provider Connectivity client: %s", err)
+	}
+
+	var hasChange bool
+	var updateOpts tenant_connection_requests.UpdateOpts
+
+	if d.HasChange("name") {
+		hasChange = true
+		updateOpts.Name = d.Get("name").(string)
+	}
+
+	if d.HasChange("description") {
+		hasChange = true
+		updateOpts.Description = d.Get("description").(string)
+	}
+
+	if d.HasChange("tags") {
+		hasChange = true
+		updateOpts.Tags = getTags(d, "tags")
+	}
+
+	if d.HasChange("name_other") {
+		hasChange = true
+		updateOpts.NameOther = d.Get("name_other").(string)
+	}
+
+	if d.HasChange("description_other") {
+		hasChange = true
+		updateOpts.DescriptionOther = d.Get("description_other").(string)
+	}
+
+	if d.HasChange("tags_other") {
+		hasChange = true
+		updateOpts.TagsOther = getTags(d, "tags_other")
+	}
+
+	if hasChange {
+		r := tenant_connection_requests.Update(client, d.Id(), updateOpts)
+		if r.Err != nil {
+			return fmt.Errorf("error updating ECL Provider Connectivity Tenant Connection Request: %s", r.Err)
+		}
+		log.Printf("[DEBUG] Tenant Connection Request has successfully updated.")
+	}
+
+	return resourceProviderConnectivityTenantConnectionRequestV2Read(d, meta)
+}
+
+func resourceProviderConnectivityTenantConnectionRequestV2Delete(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*Config)
+	client, err := config.providerConnectivityV2Client(GetRegion(d, config))
+	if err != nil {
+		return fmt.Errorf("error creating ECL Provider Connectivity client: %s", err)
+	}
+
+	if err := tenant_connection_requests.Delete(client, d.Id()).ExtractErr(); err != nil {
+		return fmt.Errorf("error deleting ECL Provider Connectivity Tenant Connection Request: %s", err)
+	}
+
+	stateConf := &resource.StateChangeConf{
+		Target:     []string{"DELETED"},
```

Comment:
You use upper case("DELETED") here but lower case("registered") in create function. Please make Consistency in codes.

Reply:
I'll do it.